phys/module_ra_rrtmg_lw.F

```diff
@@ -11740,15 +11742,14 @@ SUBROUTINE RRTMG_LWRAD(                                        &
 
 !ccc To add time-varying trace gases (CO2, N2O and CH4). Read the conc.  from file
 ! then interpolate to date of run.
-#ifdef CLWRFGHG
 ! CLWRF-UC June.09
-      REAL(8)                                      :: co2, n2o, ch4, cfc11, cfc12
-#else
+      REAL(8), SAVE             :: clco2, cln2o, clch4, clcfc11, clcfc12
```

Comment:
@smileMchen @davegill Compared to the original GHG code, maybe SAVE should not be used here?

Reply:
@weiwangncar 
Wei,
I have deleted all the 'save' statement in the codes. I believe this could at least  be one reason for the bit-identical failure. I am compiling the code manually now. Once it is done, I will push to GitHub.